tests/functional/wdio.conf.js

```diff
@@ -18,7 +18,12 @@ exports.config = {
   //
   specs: ["./tests/functional/spec/**/*.js"],
   suites: {
-    timeout_modal: ["./tests/functional/spec/timeout/*.spec.js"],
+    timeout_modal_expired: ["./tests/functional/spec/timeout/timeout_modal_expired.spec.js"],
+    timeout_modal_extended: ["./tests/functional/spec/timeout/timeout_modal_extended.spec.js"],
+    timeout_modal_extended_new_window: ["./tests/functional/spec/timeout/timeout_modal_extended_new_window.spec.js"],
+    timeout_modal_post_submission_expired: ["./tests/functional/spec/timeout/timeout_modal_post_submission_expired.spec.js"],
+    timeout_modal_post_submission_extended: ["./tests/functional/spec/timeout/timeout_modal_post_submission_extended.spec.js"],
+    timeout_modal_post_submission_extended_new_window: ["./tests/functional/spec/timeout/timeout_modal_post_submission_extended_new_window.spec.js"],
```

Comment:
The only problem is these will take longer, it will be consecutive 2 minutes waiting time times 3 plus test time so more than longest test we have now. I've tested these as 3 per spec before and there was no improvement compared to all of them in one spec if I recall correctly.

Reply:
All of it together takes ~10 minutes, so how does splitting them into 3 also take ~10 minutes?.

Reply:
it takes 6 when you split them into 3, each of them around 6 minutes.

Reply:
Ya, which is good, given each on their own also takes 6 minutes. Split into 3, takes ~ 6 because it uses both cores. So all good then?

Reply:
Ok, got it now, thanks.